### DISCUSSION TITLE
Possible to migrate only portion of a schema/migrate based on a certain condition?

### DISCUSSION THREAD

**Discussion by mariasise** [flamingock/mongock#34] [Q&A] [answered]
This is a question about whether mongock offers a particular functionality and I'm having some trouble figuring this out based upon the examples. If I want to only take certain fields from a huge schema and migrate those, is that possible/easy to do with mongock without migrating all of the fields? An unrelated question is about shuffling around a schema (e.g. moving an inner nested field to outer), but I'm assuming this can be accomplished via a series of add and remove change units.

**Comment by dieppa** [comment, marked as answer]
Hello @mariasise ,

Mongock doesn't provide any particular feature to do that. It just provides the framework for handle the migrations by using the drivers you of your choice.

**Reply by mariasise** [reply]
Thank you for the answer. Yes as I dug in further it seems evident that the approach is more to support custom filtering implementations within the migration execution. But I suppose it did not hurt to ask. Thank you for the response.
